### DISCUSSION TITLE
Repository for Extracted Data from Third-Party (e.g., GRIP)

### DISCUSSION THREAD

**Discussion by moawnallah** [InternetHealthReport/gsoc#13] [Ideas]
Hi everyone,

I wanted to initiate a discussion regarding the placement of the extracted data from third-party sources, specifically [GRIP](http://grip.inetintel.cc.gatech.edu/). Currently, the [ihr-website repository](https://github.com/InternetHealthReport/ihr-website) seems to be primarily focused on the IHR website app and its integration with the IHR APIs. Considering this, I believe it would be beneficial to explore the possibility of integrating the extracted data into another repository (if there is any repository already found for integrating data from the third-parties).

I'd love to hear your thoughts on this matter. Please share your opinions and insights so that we can collectively decide on the best course of action. Thank you!

**Comment by moawnallah** [comment]
I suggest simplifying it for now and integrating it with the `ihr-web` repository. We can consider integrating with other repositories or our public datasets at a later stage.

**Comment by romain-fontugne** [comment]
Do you mean the code that will do the API calls for third-party datasets? or downloadin data and archiving it?

For the API calls, I agree for now we can integrate that to the ihr-website repo. If that repo get too complicated we can later split it.

**Reply by moawnallah** [reply]
Yeah, I meant the code that will do the API calls for third-party datasets
